### ISSUE TITLE
[Northrend][Storm Peaks][Quest] Battling the elements

### ISSUE THREAD

**Issue by Akkalevil** [dalaranwow/dalaran-wow#1511]
- [x] Gather Snow allows you to throw multiples snowball instead of one.
- [x] If you throw a snowball when not targeting a Revenant, it kills you.
- [x] The number of Revenants is too few (half of what is needed).
- [x] Snorri lacks the dialogue when falling asleep.
- [x] If you get attacked the snow ball aura disappears, quite annoying if a Revenant is attacking you.
- [x] Respawntimer is to high. should be around 5 seconds.

http://www.wowhead.com/quest=12967/battling-the-elements

**Comment by darkwulfv** [comment]
All of these problems still exist, though the number of revenants isn't too big of a deal; they respawn very quickly.

**Comment by Akkalevil** [comment]
Two years later, nothing has changed, exactly the same bug still present...

**Comment by budthechud** [comment]
Screenshot showing the combat log of me killing myself :(
http://i.imgur.com/FYI5BVL.jpg

**Comment by robinsch** [comment]
Issue 1, 2, 3 on PTR.

@Rushor Can you check 4 and 5, please. Thanks.

**Comment by Golganeth** [comment]
Now Snowballs can't be thrown to the Seething Revenants.

**Comment by robinsch** [comment]
Was it before or after @Lothloryen commit?

**Comment by Golganeth** [comment]
After. But that one was just correction for conditions for Fjorn / Mending Fences quest item.

**Comment by Golganeth** [comment]
1 and 3 work. 2, 4 and 5 issues still happen.

**Comment by Rushor** [comment]
alright, i will check it out @robinsch

**Comment by Rushor** [comment]
cannot confirm: "If you get attacked the snow ball aura disappears, quite annoying if a Revenant is attacking you." anymore.

**Comment by Rushor** [comment]
https://www.youtube.com/watch?v=aRwHz4r3NdQ

**Comment by Rushor** [comment]
DB/Quest: Battling the Elements

* closes https://github.com/dalaranwow/dalaran-wow/issues/1511
* commit: f23d226f73ad92656e95c33fa9690d5c22552c39
* commit: 91add17b3a1bc57b033d08906808e19cc17eed97
* snorri will now talk some stuff once he wake up:
```sql
(30124, 1, 0, 'Oh yes, now... where were we?', 12, 0, 100, 0, 0, 0, 31146, 'Snorri text - wake up'),
(30124, 1, 1, 'You say something?', 12, 0, 100, 0, 0, 0, 31145, 'Snorri text - wake up'),
(30124, 1, 2, 'Just had a little snow in my eye, there...', 12, 0, 100, 0, 0, 0, 31144, 'Snorri text - wake up'),
(30124, 1, 3, 'Huh, whassat?', 12, 0, 100, 0, 0, 0, 31143, 'Snorri text - wake up'),
(30124, 1, 4, 'Ahh, at last... Time for a little break.', 12, 0, 100, 0, 0, 0, 31147, 'Snorri text - wake up');
```
* Added 5 spawns and reduced the respawntimer to 5 seconds. 

The issue is fixed on the PTR Realm. The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Please see and watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Everything poinded above is fixed on ptr, however there are few missing elements:
 - npc http://wotlk.openwow.com/npc=30120 should attack npc http://wotlk.openwow.com/npc=30121 on spawn
 - npc http://wotlk.openwow.com/npc=30120 should have pooling of 3 max up at all given times and when 1 is killed other should spawn instantly

There are more issues related to this report that have been "partly" fixed in #4631
